pkgs/development/python-modules/pytask/default.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  hatchling,
+  hatch-vcs,
+  attrs,
+  click,
+  click-default-group,
+  networkx,
+  optree,
+  packaging,
+  pluggy,
+  rich,
+  sqlalchemy,
+  universal-pathlib,
+  pytestCheckHook,
+  deepdiff,
+  nbmake,
+  pexpect,
+  pytest-cov,
+  pytest-xdist,
+  syrupy,
+  aiohttp,
+  git,
+}:
+buildPythonPackage rec {
+  pname = "pytask";
+  version = "0.5.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "pytask-dev";
+    repo = pname;
```

```suggestion
    repo = "pytask";
```
`pname` should not be used here since `repo` and `pname` are semantically different.